packaging/linux/build_binaries.sh

```diff
@@ -92,6 +92,12 @@ build_one_architecture() {
   cp "$here/crypto_squirrel.txt" "$layout_dir/opt/keybase/"
 
   # Build the kbfsfuse binary. Currently, this always builds from master.
+  # Determine the LD flags. Note that PrereleaseBuild is a different symbol in KBFS.
+  if [ "$mode" != "production" ] ; then
+    # The non-production build number is everything in the version after the hyphen.
+    build_number="$(echo -n "$version" | sed 's/.*-//')"
+    ldflags="-X github.com/keybase/kbfs/libkbfs.PrereleaseBuild=$build_number"
+  fi
```

Comment:
It looks like this mostly duplicates another if-block at the top of the file. Maybe we can move this code up there? I think the simplest thing would be to define `client_ldflags` and `kbfs_ldflags` instead of just `ldflags`, and then all that's needed down here would be a variable name change at line 104.

Also I think the `-n` flag was unnecessary back when I wrote that echo command :)